applications/ContactStructuralMechanicsApplication/custom_processes/alm_variables_calculation_process.cpp

```diff
@@ -16,92 +16,90 @@
 // Project includes
 #include "utilities/math_utils.h"
 #include "contact_structural_mechanics_application_variables.h"
-#include "custom_utilities/contact_utilities.h"
 #include "custom_processes/alm_variables_calculation_process.h"
 
 namespace Kratos
 {
-    void ALMVariablesCalculationProcess::Execute()
+void ALMVariablesCalculationProcess::Execute()
+{
+    KRATOS_TRY
+    
+    /* We compute the penalty factor */
+    
+    // We initialize the mean values
+    double mean_young_modulus_slave  = 0.0;
+    double mean_nodal_h_slave        = 0.0;
+    double mean_young_modulus_master = 0.0;
+    double mean_nodal_h_master       = 0.0;
+    
+    // We initialize the total areas and volumes
+    double total_volume_slave  = 0.0;
+    double total_area_slave    = 0.0;
+    double total_volume_master = 0.0;
+    double total_area_master   = 0.0;
+    
+    // Now we iterate over the conditions to calculate the nodal area
+    ConditionsArrayType& conditions_array = mrThisModelPart.Conditions();
+    
+#ifdef _OPENMP
```

Comment:
in principle you do not need this ifdef. the pragma will be ignored if you don't have openmp

Reply:
Ok, I will remove them, I had a compilation problem with a Clandg version without OMP and I added that